src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,207 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
 					?.SendFlyoutItemsChanged();
 		}
 
-		/// <summary>Bindable property for attached property <c>TabBarIsVisible</c>.</summary>
+		/// <summary>
+		/// Manages the bottom tab bar visibility.
+		/// The tab bar and tabs are visible in <see cref = "Shell" /> applications by default. 
+		/// </summary>
 		public static readonly BindableProperty TabBarIsVisibleProperty =
 			BindableProperty.CreateAttached("TabBarIsVisible", typeof(bool), typeof(Shell), true);
 
-		/// <summary>Bindable property for attached property <c>TitleView</c>.</summary>
+		/// <summary>
+		/// Enables any <see cref = "View" /> to be displayed in the navigation bar.
+		/// </summary>
 		public static readonly BindableProperty TitleViewProperty =
 			BindableProperty.CreateAttached("TitleView", typeof(View), typeof(Shell), null, propertyChanged: OnTitleViewChanged);
 
-		/// <summary>Bindable property for attached property <c>MenuItemTemplate</c>.</summary>
+		/// <summary>
+		/// Customizes the appearance of each <see cref = "MenuItem" />.
+		/// </summary>
 		public static readonly BindableProperty MenuItemTemplateProperty =
 			BindableProperty.CreateAttached(nameof(MenuItemTemplate), typeof(DataTemplate), typeof(Shell), null, BindingMode.OneTime);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetMenuItemTemplate']/Docs/*" />
-		public static DataTemplate GetMenuItemTemplate(BindableObject obj) => (DataTemplate)obj.GetValue(MenuItemTemplateProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetMenuItemTemplate']/Docs/*" />
-		public static void SetMenuItemTemplate(BindableObject obj, DataTemplate menuItemTemplate) => obj.SetValue(MenuItemTemplateProperty, menuItemTemplate);
-
-		/// <summary>Bindable property for attached property <c>ItemTemplate</c>.</summary>
-		public static readonly BindableProperty ItemTemplateProperty =
+        /// <summary>
+        /// Gets the <see cref = "DataTemplate" /> applied to <see cref = "MenuItem" /> objects in the MenuItems collection.
+        /// </summary>
+        /// <param name="obj">The object that sets the DataTemplate to be applied to the MenuItem objects.</param>
+        /// <returns>The DataTemplate applied to MenuItem objects.</returns>
+        public static DataTemplate GetMenuItemTemplate(BindableObject obj) => (DataTemplate)obj.GetValue(MenuItemTemplateProperty);
+
+        /// <summary>
+        /// Sets the <see cref = "DataTemplate" /> applied to <see cref = "MenuItem" /> objects in the MenuItems collection.
+        /// Shell provides the Text and IconImageSource properties to the BindingContext of the <see cref = "MenuItemTemplate" />. 
+        /// You can also use Title in place of Text and Icon in place of IconImageSource which will let you reuse the same template for menu items and flyout items.
```

Comment:
Avoid using "you..." in sentences, and rather state what can be done. I also think that a bit of the wording is confusing. 
Consider this suggestion: 

```suggestion
        /// Title can be used instead of Text, and Icon instead of IconImageSource. This allows reuse of the same template for menu items and flyout items.
```

Reply:
The extra sentences can go in the `remarks` section, and don't forget to add the crefs!

Reply:
Added `remarks `and `crefs`.